protocols/identify/src/lib.rs

```diff
@@ -62,9 +63,15 @@ impl MisbehaveResult {
 }
 
 /// The trait to communicate with underlying peer storage
-pub trait AddrManager: Clone + Send {
+pub trait Callback: Clone + Send {
+    /// Add local init listen address
+    fn init_local_listen_addrs(&mut self, addrs: Vec<Multiaddr>);
```

Comment:
unnecessary callbacks

Reply:
`Network` can already get `listen_addrs` from configuration and event Stream, So it is unnecessary to get addresses from identify_protocol.